app/views/admin/registrations/index.html.haml

```diff
@@ -44,24 +44,14 @@
               %td
                 = link_to 'Questions','#', class: 'btn btn-success question-btn', 'data-id' => index, 'data-name' => registration.name
             %td
+              = check_box_tag @conference.short_title, registration.id, true, class: 'switch-checkbox', method: :patch,
+                data: { size: "large",
```

Comment:
Try 
```
size: "small"
```
And as the button doesn't belong to the button group anymore I would consider an additional table column.